### DISCUSSION TITLE
Clear definition of "low carbon" and "renewables" to FAQ

### DISCUSSION THREAD

**Discussion by jaakla** [electricitymaps/electricitymaps-contrib#2796] [General]
I get that biomass can be classified as kind of "renewable" (but consider that regrowing trees can take 50+ years in higher latitudes), but burning it for energy or electricity produces much more CO₂ per KWh than many fossils, e.g. natural gas. In this sense biomass should be set as "high carbon".  
From the numbers it seems that you simply classify all renewables as low carbon. I know it is a common approach, but still IMHO debatable and greenwashy practise. Please you add to FAQ what exactly you classify as low carbon and renewables

**Comment by chalg** [comment]
Agree, and is transportation, for example, shipping it from the US to Europe included in emissions?

**Comment by jarek** [comment]
> burning it for energy or electricity produces much more CO₂ per KWh than many fossils, e.g. natural gas. In this sense biomass should be set as "high carbon".

According to the IPCC values which we use, the median carbon intensity (gCO2eq/kWh) of electricity generated from biomass is about half of that generated from natural gas. See https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf PDF-page 1335. The mean/median/max values for biomass are 130 / 230 / 420 compared to 410 / 490 / 650 for combined-cycle natural gas. The biomass figure has a note "Direct emissions from biomass combustion at the power plant are positive and significant, but should be seen in connection with the CO2 absorbed by growing plants." Generally the position of Electricity Map has been that they want to use peer-reviewed values for carbon intensity and the IPCC values are used as a widely-reviewed baseline.

> is transportation, for example, shipping it from the US to Europe included in emissions?

IPCC figures we use include lifecycle emissions. E.g. for nuclear this includes uranium mining and concrete embodied in the plants. For biomass, most of the overall carbon intensity value does come from "infrastructure & supply chain emissions".

---

You may also be interested in previous discussions in https://github.com/tmrowco/electricitymap-contrib/issues/738 "Refine GHG emission factors by regions"

---

On this topic, I think that our readme used to have a link the IPCC table that we use for default carbon intensity of the different fuels, but I can't find it now. Has it gone missing? I _think_ the PDF link is https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf ?

**Reply by ofenaus** [reply]
I think @jaakla is correct here and you are in fact not using the IPCC values.
According to the quoted document the direct emissions for biomass (dedicated) are not 0. Instead the footnote says they are "significant" - which is correct, actually they are higher than the emissions of coal. If they were 0, the IPCC would have used this value like they did for hydropower  or geothermal. But they are not.

I strongly recommend to have a look into the most recent scientific papers on this topic, e.g.

https://www.researchgate.net/publication/357375767_Primeval_natural_and_commercial_forests_in_the_context_of_biodiversity_and_climate_protection_Part_2_The_narrative_of_the_climate_neutrality_of_wood_as_a_resource

Especially the blue box on page 33 outlines the issue of non-existant millions tons of carbon dioxide from woody biomass.
More sources:
https://iopscience.iop.org/article/10.1088/1748-9326/aaa512/pdf
https://publications.jrc.ec.europa.eu/repository/handle/JRC122719
https://www.wwf.eu/?2128466%2F500-scientists-tell-EU-to-end-tree-burning-for-energy

Currently electricitymap is not providing accurate data and is supporting greenwashing of dirty biomass.

https://twitter.com/BeyondBurning/status/1470364778631684108

Please take action.

**Reply by ofenaus** [reply]
Furthermore:

The IPCC models you are using did NOT assess use of forest-derived wood pellets for electricity:

https://www.nrdc.org/experts/sasha-stashwick/what-ipcc-really-says-forest-biomass-climate-change

This becomes VERY clear if you have a look at the FAQ published by the NGGIP:

https://www.ipcc-nggip.iges.or.jp/faq/faq.html

Q2-10:

The IPCC guidelines provide the simplifying assumption (Tier 1 method) regarding CO2 emissions and removals associated with annual biomass (e.g., corn, colza) that over the course of a year, the CO2 emissions from the combustion/oxidation/decay of annual biomass are balanced by carbon uptake prior to harvest, within the uncertainties of the estimates, so the net emission is zero.

This assumption is ONLY valid for annual biomass. Woody biomass has never been declared net-zero by the IPCC. NEt-zero biomass is only an accounting rule, not a scientific fact:

https://ens.dk/sites/ens.dk/files/Bioenergi/biomasseanalyse_final_ren_eng.pdf

**Comment by Kongkille** [comment]
@jarek we added this section in the wiki. Would be happy to expand it if you have any input whatsoever :) 
https://github.com/tmrowco/electricitymap-contrib/wiki/Emission-factors

I converted this to a discussion, we can just convert it back if we find a specific change we want to do
